docs/presidio_V2.md

```diff
@@ -0,0 +1,167 @@
+# Presidio Revamp (aka V2)
+
+As of March 2021, Presidio had undergo a revamp to a new version refereed to as **V2**.
+
+The main changes introduced in **V2** are:
+
+1. gRPC replaced with HTTP to allow more customizable APIs and easier debugging
+2. Presidio Anonymizer is Python based and pip installable.
+3. Focus on the Analyzer and Anonymizer services. Other services are deprecated and potentially be migrated over time to **V2** with the help of the community.

```

Comment:
```suggestion
3. Focus on the Analyzer and Anonymizer services. 

Other services are deprecated and potentially be migrated over time to **V2** with the help of the community.

```

Reply:
not sure this should be a seperate comment

Reply:
it's not separate comment, just a new line :)

Reply:
but it's a nit